packages/roosterjs-editor-plugins/lib/plugins/HyperLink/HyperLink.ts

```diff
@@ -67,6 +68,34 @@ export default class HyperLink implements EditorPlugin {
      * @param event PluginEvent object
      */
     public onPluginEvent(event: PluginEvent): void {
+        if (
+            event.eventType == PluginEventType.MouseUp ||
+            (event.eventType == PluginEventType.KeyUp &&
+                event.rawEvent.which >= Keys.PAGEUP &&
+                event.rawEvent.which <= Keys.DOWN)
```

Comment:
1. change a link href
2. without click or move cursor, save the result directly

Then the original link without changed will be saved

Reply:
Ah yeah I see the issue, thanks for catching this. Working on fixing it.

Reply:
Should be fixed with latest commit - let me know if you think of any issues with it